JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -159,6 +158,13 @@ - (void)jsq_configureMessagesViewController
     self.inputToolbar.contentView.textView.delegate = self;
     self.inputToolbar.contentView.textView.font = [UIFont preferredFontForTextStyle:UIFontTextStyleBody];
     [self.inputToolbar removeFromSuperview];
+    
+    self.recordView = MEGARecordView.recordView;
+    [self.view addSubview:self.recordView];
+    [self.recordView autoSetDimensionsToSize:CGSizeMake(180, 180)];
+    [self.recordView autoCenterInSuperview];
```

Comment:
This should center the recordView but is not working, the view is placed a bit lower in the Y axis. I guess the superview is not covering the full screen, could you check it?

Reply:
Do you mean that we should physically centre this hub?
If so, I can add some offset for it.

Reply:
I mean the position of the hud is not centered as in designs. Not a big problem but neither difficult to do I think.

![hud recording audio dev vs design](https://user-images.githubusercontent.com/6225136/71248276-bd48ad00-231a-11ea-9eb3-dbdffb972a61.png)

Reply:
got it.